stubs/python-jose/jose/jws.pyi

```diff
@@ -4,21 +4,21 @@ from typing import Any
 from .backends.base import Key
 
 def sign(
-    payload: str | Mapping[str, Any],
+    payload: bytes | Mapping[str, Any],
     # Internally it's passed down to jwk.construct(), which explicitly checks for
     # key as dict instance, instead of a Mapping
-    key: str | dict[str, Any] | Key,
+    key: str | bytes | dict[str, Any] | Key,
     headers: Mapping[str, Any] | None = ...,
     algorithm: str = ...,
 ) -> str: ...
 def verify(
-    token: str,
-    key: str | Mapping[str, Any] | Key,
+    token: str | bytes,
+    key: str | bytes | Mapping[str, Any] | Key,
     # Callers of this function, like jwt.decode(), and functions called internally,
     # like jws._verify_signature(), use and accept algorithms=None
     algorithms: str | Container[str] | None,
     verify: bool = ...,
-) -> str: ...
+) -> bytes: ...
```

Comment:
It is returned from `base64url_decode`, which returns `bytes`.
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L70
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L193
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/utils.py#L66